docs/source/en/api/pipelines/mochi.md

```diff
@@ -25,6 +25,50 @@ Make sure to check out the Schedulers [guide](../../using-diffusers/schedulers.m
 
 </Tip>
 
+## Generating videos with Mochi-1 Preview
+
+The following example will download the full precision `mochi-1-preview` weights and produce the highest quality results but will require at least 42GB VRAM to run.   
+
+```python
+import torch
+from diffusers import MochiPipeline
+from diffusers.utils import export_to_video
+
+pipe = MochiPipeline.from_pretrained("genmo/mochi-1-preview")
+
+# Enable memory savings
+pipe.enable_model_cpu_offload()
+pipe.enable_vae_tiling()
+
+prompt = "Close-up of a chameleon's eye, with its scaly skin changing color. Ultra high resolution 4k."
+
+with torch.autocast("cuda", torch.bfloat16, cache_enabled=False):
+      frames = pipe(prompt, num_frames=84).frames[0]
+
+export_to_video(frames, "mochi.mp4", fps=30)
+```
+
+## Using a lower precision variant to save memory
+
+The following example will use the `bfloat16` variant of the model and requires 22GB VRAM to run. There is a slight drop in the quality of the generated video as a result.
```

Comment:
should bf16 be used? fp16 seems to [have better precision](https://github.com/huggingface/diffusers/pull/9769#issuecomment-2466593664), unless the autocasted weights compute in fp32 and somehow provide better precision?

Reply:
We followed the official repo, which cast to bf16.  
https://github.com/genmoai/models/blob/1a96b5df0c83018fcbe7aebbbb1b34781e179921/src/genmo/mochi_preview/pipelines.py#L344

Reply:
I am indeed, though I'm also doing mixed weight precision between float8 and f16. It could be beneficial to test further.

Reply:
We can keep it to bfloat16 for now (as done in this PR) following the official recommendations. Then @Ednaordinary you could maybe help us opening a PR to the docs including your findings? WDYT?

Reply:
Sure, that works for the moment